workflow/exp/exp.ens_conus12km

```diff
@@ -42,7 +42,7 @@ export LBC_GROUP_TOTAL_NUM=1
 export RETRO_PERIOD="2024052700-2024052800"
 export CYCLEDEF_IC_STEP_HRS=6
 export CYCLEDEF_LBC_STEP_HRS=6
-export COLDSTART_CYCS="00 12"   # cycles to cold start from external models
+export COLDSTART_CYCS="00 06 12 18"   # cycles to cold start from external models
```

Comment:
Could you provide more details about why we want to cold start 4 times each day? Does it include ensemble DA?

Reply:
For the ensemble without DA yet, we don't want to run free ensemble forecast for too long. That is why we initialize the ensemble 4 times a day to minimize the long ensemble forecast.

Reply:
Got it. Thanks!